src/core/tests/FileSystemTestSuite.cpp

```diff
@@ -29,62 +33,155 @@ namespace {
                        return wPath == wFileName || wPath == wFileName2;
                      });
 }
+
+void createFile(const std::string& filePath, std::size_t sizeInBytes = 4096) {
```

Comment:
You can use the following `src/core/utility/file/scopedTemporaryFile/ScopedTemporaryFile.hpp`

Reply:
I agree to fulfill DRY principle.